spouse/spouse_demo.py

```diff
@@ -0,0 +1,363 @@
+# %% [markdown]
+# # Detecting spouse mentions in sentences
+
+# %% [markdown]
+# We will walk through an example text classification task to explore how Snorkel can be used for Information Extraction.
+# ## Classification Task
+# <img src="imgs/sentence.jpg" width="700px;">
```

Comment:
fyi—this breaks in my browser preview: 
![image](https://user-images.githubusercontent.com/5819303/62608404-a0a06100-b8b4-11e9-9f25-0d2ae04caa7c.png)

Reply:
Fixed. The image was empty for some reason.